File: MatMan13_16x64/MatMan_Lib/AppCode/Application Server/ClearValues.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace iiiwave.MatManLib
{
    class ClearValues
    {
        public static void ClearPreviousRun()
	    {
		    MatManFunctionCollection.GetObject().Clear();
		    MatManFunctionCollection.GetObject().TotalFunctionsAddedToQueue = 0;

		    SAPRequest.GetObject().TotalProcessedBySAP                      = 0;
		    SAPRequest.ReturnValuesList.Clear();

		    //MatManCalcEngine.GetObject().CurrentFunctionsByCellAddress.Clear();

		    //MatManCalcEngine.GetObject().ConnectDataCount                   = 0;
		    //MatManCalcEngine.GetObject().TopicCount                         = 0;
		    //MatManCalcEngine.GetObject().UserInitiatedCalc                  = true;
	    }
    }
}

File: MatMan13_16x64/MatMan_VB/AppCode/ApplicationServer/ClearValues.vb


Public Class ClearValues
    Public Shared Sub ClearPreviousRun()
	    'MatManFunctionCollection.GetObject().Clear()
	    'MatManFunctionCollection.GetObject().TotalFunctionsAddedToQueue = 0

	    'SAPRequest.GetObject().TotalProcessedBySAP = 0
	    'SAPRequest.ReturnValuesList.Clear()

	    MatManCalcEngine.CurrentFunctionsByCellAddress.Clear()

	    MatManCalcEngine.ConnectDataCount = 0
	    MatManCalcEngine.TopicCount = 0
	    MatManCalcEngine.UserInitiatedCalc = True
    End Sub
End Class
